src/components/Checkbox/Checkbox.styl

```diff
@@ -0,0 +1,23 @@
+.container {
+	display: inline-block;
+	position: relative;
+
+	.input {
+		position: absolute;
+		clip: rect(0,0,0,0);
```

Comment:
`clip` is [deprecated](https://developer.mozilla.org/en/docs/Web/CSS/clip#Browser_Compatibility). consider hiding input with some other technique